history/timeline.txt

```diff
@@ -223,3 +223,37 @@ Mar 23:
 This is the last meeting with Hans, as he would leave for training in the next week.
 The elections were also postponed to the next meeting, as all of the parties were not ready for them.
 
+April 6, 2024:
+This was the first meeting without hans, and Itay presented a video he made for him showing pictures of him and playing the song "Ze shir preda".
+The elections were held in this meeting.
+
+Parties:
+“Yesh Kofta” (Ido Winter)
+“Koftai” (Itay Winter)
+“Shamen” (Dor Winter and Maor Winter)
+“Guftait” (Gila)
+
+ due to the new direct elections law, candidates compete for the role of prime minister. The candidates were:
+
+Itay Winter
+Ido Winter
+Dor Winter
+
+Election Results:
+
+Parliament:
+Ido: 10 mandates
+Itay: 20 mandates
+Dor: 40 mandates
+Gila: 40 mandates
+
+Prime Minister:
+Itay: 5 votes
+Dor: 5 votes
+Ido: 1 vote
+
+Given the new direct election law, if there’s a tie between two candidates, a second round of voting will take place in the next meeting.
+Therefore Itay and Dor will compete for the seciond time in the next meeting for the role of prime minister.
+
+
+
```

Comment:
- [ ] Write about Ido imitating Jackie Sabag by calling the citizens "ungrateful"